internal/golada/cmd/generate.go

```diff
@@ -120,7 +122,7 @@ func Generate(path string, parser annotation.Parser, logger logger.Logger) {
 }
 
 func init() {
-	generateCommand.PersistentFlags().StringVar(&parserType, "parser", "property", "parser [csv,property]")
+	generateCommand.PersistentFlags().StringVarP(&parserType, "parser", "p", "property", "parser [csv,property,build-tag]")
```

Comment:
The description field does not really help me understand the flag. I think a little bit more text would be helpful.

Reply:
The field is labelled usage not description. As of right now I simply provide a regex like approach to the possible use-cases. Is this supposed to be a description ?

Reply:
Right, usage, not description. It is suppose to help a user to understand how to use it. Maybe a sentence based on what you just wrote in the comment here on how to use that flag.

Reply:
Alright, I added a really basic description.